astropy/stats/sigma_clipping.py

```diff
@@ -180,6 +180,7 @@ def sigma_clipped_stats(data, mask=None, mask_val=None, sigma=3.0, iters=None):
         image.
     """
 
+    data = np.asanyarray(data)
```

Comment:
Is this actually necessary? In https://github.com/larrybradley/astropy/blob/stats/astropy/stats/sigma_clipping.py#L127 a masked array is created; I would think this would take care.  Of course, the best way to ensure this works is to add a test! (with, e.g., a list as input).